openai/tests/test_client.py

```diff
@@ -10,11 +55,36 @@ def clear_oai_module(monkeypatch: pytest.MonkeyPatch):
     monkeypatch.setattr(openai, 'api_key', None)
     monkeypatch.setattr(openai, 'api_type', "open_ai")
     monkeypatch.setattr(openai, 'api_version', None)
+    monkeypatch.setattr(openai, 'organization', None)
 
 def setup_oai_module(monkeypatch: pytest.MonkeyPatch, **kwargs):
     for n, v in kwargs.items():
         monkeypatch.setattr(openai, n, v)
-        
+
+@pytest.fixture
+def embedding_response():
```

Comment:
I think we can get rid of this fixture. I was (ab)using it incorrectly in the `assert`s. There is no reason for the mocked response to adhere to the actual output, so we can just echo the parameters instead...

Reply:
done